### PR TITLE
Compact messages sent to players, improve grammar

### PR DESCRIPTION
**Pull request by DoNotSpamPls**
Referencing #120 
I have only tested this PR on Spigot, but it should work on BungeeCord and Sponge too, since I've done the same changes to them.

P.S. Wow, only 3 PRs to the repository so far? I really thought they were more.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Eclipse stuff
-/.classpath
```
Comment (games647) [inline .gitignore]: Those aren't folders, but they reference only the contents inside the root folder of the project which doesn't make sense neither. I didn't noticed that earlier thank you very much for pointing that out.

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
-/.project
+.classpath
+.project
 /.settings
```
Comment (games647) [inline .gitignore]: Could you remove the first / from other entries too? So to
```
.settings/
target/
```
and more

**Comment by games647** [comment]
Thank you very much. It looks great.

>P.S. Wow, only 3 PRs to the repository so far? I really thought they were more.

ChangeSkin doesn't have the same popularity in comparison to other skin plugins like SkinsRestorer. Furthermore many people are not familiar with ProtocolLib, but besides that ChangeSkin code is not really the best. It is complex and contains some duplicated logic (because of the three different platforms we are targeting).

**Comment by DoNotSpamPls** [comment]
> Could you remove the first / from other entries too?

Done.

> ChangeSkin doesn't have the same popularity in comparison to other skin plugins like SkinsRestorer. Furthermore many people are not familiar with ProtocolLib, but besides that ChangeSkin code is not really the best. It is complex and contains some duplicated logic (because of the three different platforms we are targeting).
Nevertheless I'm happy to see new contributors joining the project although it's complicated code.

The codebase of this plugin is already like 5x better than SkinsRestoer's one (first-hand experience), even with some complexity and duplicated logic. I think it deserves a lot more attention, and I'm more than happy to contribute to the plugin's development 😃

**Pull request merged**